### DISCUSSION TITLE
Cúal es la forma correcta de subir la Kata 2?

### DISCUSSION THREAD

**Discussion by ChanLuis** [LaunchX-InnovaccionVirtual/CursoIntroPython#88] [Q&A]
Esta bien esto que hice de subir la Kata 2 como un recorte de SS

![Subir katas](https://user-images.githubusercontent.com/99062113/153504452-d07d0927-3e3c-45ef-805d-b9e20a50bbf1.jpg)

**Comment by OscarPedroza** [comment]
Yo lo que realicé fue, en un notebook ir colocando las capturas de los comandos.

**Reply by RodolfoMorquecho** [reply]
Disculpa mi ignorancia, pero el notebook lo hiciste en visual? crees que haya problema si lo voy haciéndolo con capturas de pantalla en word?

**Comment by doguedogue** [comment]
Así es, yo creo que resolviendo los notebooks, y al guardarlos indica que lo ejecutaste y cuanto tiempo demoró ej. 1s, 2s

**Comment by GabrielVillagran** [comment]
Hola, nos comentaron que la kata 2 también se puede subir con capturas de pantallas en un pdf por si se te hace más sencillo, en mi caso fui describiendo paso a paso como lo hize con capturas de pantallas
